pom.xml

```diff
@@ -269,6 +237,13 @@
                 <artifactId>duplicate-finder-maven-plugin</artifactId>
             </plugin>
         </plugins>
+        <extensions>
+            <extension>
+                <groupId>org.apache.maven.extensions</groupId>
+                <artifactId>maven-build-cache-extension</artifactId>
+                <version>1.0.0</version>
+            </extension>
+        </extensions>
```

Comment:
This does not seem to be related to the current feature. If I were the repo owner, I'd prefer a separate PR (and discussion).

Reply:
Nope is just for me during development.